### DISCUSSION TITLE
hardhat problem when testing

### DISCUSSION THREAD

**Discussion by mohamedtb20** [NomicFoundation/hardhat#2615] [General]
****
C:\Users\DELL\nft_marketplace>npx hardhat test
Error HH8: There's one or more errors in your config file:

  * Invalid account: #0 for network: mumbai - Expected string, received `function`
  * Invalid account: #0 for network: mainnet - Expected string, received function

To learn more about Hardhat's configuration, please go to https://hardhat.org/config/``

**Comment by alcuadrado** [comment]
Please, take a look at the config values and double-check your config.
